conf/messages.cy

```diff
@@ -7,45 +55,95 @@ hasEclReference.No = Na
 hasEclReference.divider = neu
 hasEclReference.Unknown = Nid wyf yn gwybod beth yw fy nghyfeirnod ECL
 hasEclReference.error.required = Dewiswch os oes gennych gyfeirnod ECL
+
 registration.title = Mae angen i chi gofrestru ar gyfer yr Ardoll Troseddau Economaidd (ECL)
 registration.heading = Mae angen i chi gofrestru ar gyfer yr Ardoll Troseddau Economaidd (ECL)
 registration.p1 = Mae angen i chi gofrestru ar gyfer yr Ardoll Troseddau Economaidd (ECL).
 registration.register = Cofrestru ar gyfer yr Ardoll Troseddau Economaidd
+
 findEclReference.title = Sut i ddod o hyd i’ch cyfeirnod Ardoll Troseddau Economaidd
 findEclReference.heading = Sut i ddod o hyd i’ch cyfeirnod Ardoll Troseddau Economaidd
 findEclReference.p1 = Gallwch ddod o hyd i’ch cyfeirnod ECL yn y llythyr a gawsoch gan CThEF, neu yn yr e-bost cadarnhau a gawsoch pan wnaethoch gofrestru.
 findEclReference.p2 = Os na allwch ddod o hyd i’ch cyfeirnod ECL, bydd angen i chi gysylltu â CThEF.
+
 eclReference.title = Eich cyfeirnod Ardoll Troseddau Economaidd (ECL)
 eclReference.heading = Eich cyfeirnod Ardoll Troseddau Economaidd (ECL)
 eclReference.p1 = Gallwch ddod o hyd i’ch cyfeirnod ECL yn y llythyr a gawsoch gan CThEF, neu yn yr e-bost cadarnhau a gawsoch pan wnaethoch gofrestru.
 eclReference.hint = Er enghraifft, XMECL0000000001
 eclReference.error.required = Nodwch gyfeirnod, enghraifft XMECL0000000001
 eclReference.error.invalid = Nodwch gyfeirnod dilys, er enghraifft XMECL0000000001
+
 eclRegistrationDate.title = Eich dyddiad cofrestru ECL
 eclRegistrationDate.heading = Eich dyddiad cofrestru ECL
 eclRegistrationDate.p1 = Rhowch wybod i ni’r dyddiad y gwnaethoch gofrestru ar gyfer ECL. Gallwch ddod o hyd i hwn yn y llythyr a gawsoch gan CThEF, neu yn yr e-bost cadarnhau a gawsoch pan wnaethoch gofrestru.
 eclRegistrationDate.hint = Er enghraifft, 13 9 2023
 eclRegistrationDate.error.notWithinRange = Mae’n rhaid i’r dyddiad fod rhwng 1 Ebrill 2022 a heddiw
-detailsDoNotMatch.title = Nid yw’ch manylion yn cyd-fynd â’n cofnodion
-detailsDoNotMatch.heading = Nid yw’ch manylion yn cyd-fynd â’n cofnodion
-detailsDoNotMatch.p1 = Nid yw’r manylion yr ydych wedi’u rhoi yn cyd-fynd â’n cofnodion.
-detailsDoNotMatch.p2 = Edrychwch ar y llythyr a gawsoch gan CThEF, neu’r e-bost a gawsoch pan wnaethoch gofrestru ar gyfer yr Ardoll Troseddau Economaidd (ECL).
```

Comment:
Why is this removed?

Reply:
Whoever did the previous Welsh translations duplicated those message keys. Seee lines 39 to 42. You can't have duplicate keys in a messages file.